pkg/kubelet/kuberuntime/kuberuntime_gc.go

```diff
@@ -121,6 +121,32 @@ func (cgc *containerGC) enforceMaxContainersPerEvictUnit(evictUnits containersBy
 	}
 }
 
+// enforceMaxSurvivalAgePerEvictUnit enforces MaxSurvivalAge for each evictUnit.
+func (cgc *containerGC) enforceMaxSurvivalAgePerEvictUnit(evictUnits containersByEvictUnit, maxSurvivalAge time.Duration) {
+	oldestGCTime := time.Now().Add(-maxSurvivalAge)
+	for key := range evictUnits {
+		containers := evictUnits[key]
+		for i := 0; i < len(containers); i++ {
+			if oldestGCTime.After(containers[i].createTime) {
```

Comment:
wrong logic?   it should be something like  "oldestGCTime.After(containers[i].exitTime)",   not "createTime".

Reply:
It would make code a bit easier to read if you continue the loop under negated condition (less indentation).